README.md

```diff
@@ -159,6 +159,8 @@ The below table lists the available templates and the CSS framework they corresp
 | `@BabDevPagerfantaBundle/twitter_bootstrap3.html.twig` | [Bootstrap](https://getbootstrap.com) (version 3.x)  |
 | `@BabDevPagerfantaBundle/twitter_bootstrap4.html.twig` | [Bootstrap](https://getbootstrap.com) (version 4.x)  |
 
+Labels of Previous and Next buttons are localizable in all of these Twig templates.
+
 When rendering a Twig view, the following options are passed into the template for use:
```

Comment:
I wonder if it actually makes much sense to list all these variables available to the template, now that blocks are just HTML markup with a couple variables each.

Reply:
I'd keep it.  If someone decides to override the base `pager_widget` or `pager` blocks then they'll want to be able to reference these.  And if nothing else it's good internal documentation.

Reply:
`pager` is actually the only block that uses all these variables. Perhaps the info about simple customizations should go a bit upwards then, since in most cases `pager` won't be overridden.
I tried to edit the readme again, but I just couldn't make it sound good, so I'll leave this up to you. :)

Reply:
I'll iterate over it.  There's probably a good way to steer folks to "override the small blocks and avoid overriding the big `{{ pager }}` block, but if you really want to do that here's what's available to you".